RELEASE.md

```diff
@@ -15,18 +15,23 @@ Please follow the established format:
 - Introduce `behaviour` prop object with `reFocus` prop (#2161)
 
 ## Bug fixes and other changes
-- Improve `kedro viz build` usage documentation (#2126)
-- Fix unserializable parameters value (#2122)
-- Replace `watchgod` library with `watchfiles` and improve autoreload file watching filter (#2134)
-- Display full dataset type with library prefix in metadata panel (#2136)
-- Enable SQLite WAL mode for Azure ML to fix database locking issues (#2131)
-- Replace `flake8`, `isort`, `pylint` and `black` by `ruff` (#2149)
-- Refactor `DatasetStatsHook` to avoid showing error when dataset doesn't have file size info (#2174)
+- Improve `kedro viz build` usage documentation. (#2126)
+- Fix unserializable parameters value. (#2122)
+- Replace `watchgod` library with `watchfiles` and improve autoreload file watching filter. (#2134)
+- Display full dataset type with library prefix in metadata panel. (#2136)
+- Enable SQLite WAL mode for Azure ML to fix database locking issues. (#2131)
+- Replace `flake8`, `isort`, `pylint` and `black` by `ruff`. (#2149)
```

Comment:
I always forget whether we're only putting user-facing changes in the `RELEASE.md` or everything. If the former, maybe this line should go away.

Reply:
@astrojuanlu As far as I remember, we keep everything to track for us and users.